server-side/controllers/profile.controller.js

```diff
@@ -2,16 +2,15 @@ import mongoose  from 'mongoose';
 import Profiles from '../models/profile.model.js';
 
 
-export const getAllProfiles = async (req, res, next) => {
-    try {
-        const profiles = await Profiles.find().populate('limitedWebsites.websiteId blockedSites').select('-__v');
-        res.json(profiles);
-    } catch (err) {
-        next({message:err.message,status:500})
-    }
-};
 
-  
+   export const  getAllProfiles= async (req, res) => {
+        try {
```

Comment:
Is this change related to your task?

Reply:
you wrote me in slack that it's not related to you, so why do you have this changes?